### ISSUE TITLE
small fixes needed to ASLLVD data

### ISSUE THREAD

**Issue by cneidle** [DCS-LCSR/ASL-DAI#261]
For ID 5220, the handshapes on the dominant and non-dominant hands (both start and end) need ti be reversed.  They should be the same as for the other 3 examples.  Thanks.

<img width="774" alt="Screen Shot 2020-11-20 at 1 18 09 PM" src="https://user-images.githubusercontent.com/13629122/99835877-3f48e380-2b33-11eb-94c4-5c39b02bf257.png">

fixed - CN

**Comment by cneidle** [comment]
For (K)KITCHEN   iD539 Add + in gloss:  should be "(K)KITCHEN+"

<img width="1097" alt="Screen Shot 2020-11-20 at 1 25 10 PM" src="https://user-images.githubusercontent.com/13629122/99836531-3efd1800-2b34-11eb-8109-26276e43fae2.png">

fixed - CN

**Comment by cneidle** [comment]
For ADVISE/INFLUENCE
ID 846 needs one +.     Should be ADVISE/INFLUENCE+
ID 848 needs two +'s.    Should be ADVISE/INFLUENCE++

<img width="645" alt="Screen Shot 2020-11-20 at 1 30 59 PM" src="https://user-images.githubusercontent.com/13629122/99836852-b2068e80-2b34-11eb-8273-dcb7d6a2b4f2.png">


fixed - cn

**Comment by cneidle** [comment]
For again, we need a few more plusses:

874   Should be glossed as AGAIN++  (rather than AGAIN)
876  Should be glossed  AGAIN+  (rather than AGAIN)

<img width="497" alt="Screen Shot 2020-11-20 at 1 35 01 PM" src="https://user-images.githubusercontent.com/13629122/99837225-3bb65c00-2b35-11eb-8e53-a83b4a5af72c.png">


fixed -cn

**Comment by cneidle** [comment]
ID 39158 has an extra plus:

Should be ACT* (ACT+AGENT) - rather than ACT* (ACT++AGENT)

<img width="747" alt="Screen Shot 2020-11-20 at 1 45 14 PM" src="https://user-images.githubusercontent.com/13629122/99838155-ab791680-2b36-11eb-93a3-40ae570508d9.png">


**THE CHANGES BELOW HAVE NOT BEEN MADE !**

The same change needs to be made to the other elements associated with this compound, i.e., 39159 and 39302:

<img width="1021" alt="Screen Shot 2020-11-20 at 1 40 00 PM" src="https://user-images.githubusercontent.com/13629122/99837659-f181aa80-2b35-11eb-8608-911e376971f5.png">

<img width="748" alt="Screen Shot 2020-11-20 at 1 42 07 PM" src="https://user-images.githubusercontent.com/13629122/99837892-41f90800-2b36-11eb-9ec7-6215fb928b60.png">

**Comment by cneidle** [comment]
**EVERYTHING ABOVE THIS LINE HAS BEEN FIXED - as of January 2022**

ACT+(1h)AGENT should be its own main entry.  So we should have

ACT+AGENT
- ACT+AGENT

ACT+(1h)AGENT
- ACT+(1h)AGENT

<img width="167" alt="Screen Shot 2020-11-21 at 1 30 37 PM" src="https://user-images.githubusercontent.com/13629122/99884672-fd38a400-2bfd-11eb-8c5d-4693204536fb.png">

fixed 0 cn

**Jan 2022:  I am not seeing this anywhere** 
 ACT+(1h)AGENT

**Comment by cneidle** [comment]
Could you please REMOVE all plus signs here?

<img width="590" alt="Screen Shot 2020-11-22 at 2 52 30 PM" src="https://user-images.githubusercontent.com/13629122/99915587-8d481d80-2cd2-11eb-8ce2-d337fa0f935a.png">

So 6032 and 6033 should just be "NERVOUS"

fixed - cn

**Comment by kwasiopoku** [comment]
This is complete

**Comment by cneidle** [comment]
Everything fixed EXCEPT for the 5th comment from the top.

**Comment by kwasiopoku** [comment]
This has been resolved

**Comment by cneidle** [comment]
thank you!

**Comment by cneidle** [comment]
I thought I checked ACT+(1h)AGENT to make sure it was its own entry, and it had seemed to be fixed:

ACT+(1h)AGENT should be its own main entry. So we should have

ACT+AGENT
- ACT+AGENT

ACT+(1h)AGENT
- ACT+(1h)AGENT

However, now there is a new problem:  The entry under ACT+(1h)AGENT has disappeared.
And it has also disappeared from the places where it should be occurring with parts of that compound, 
although I had previously verified that these also had been fixed.

<img width="109" alt="Screen Shot 2020-11-27 at 10 59 26 AM" src="https://user-images.githubusercontent.com/13629122/100467039-dcf66280-309f-11eb-9392-c566246d2bf9.png">

<img width="315" alt="Screen Shot 2020-11-27 at 11 02 42 AM" src="https://user-images.githubusercontent.com/13629122/100467235-2050d100-30a0-11eb-8001-63b812b0fe35.png">

<img width="172" alt="Screen Shot 2020-11-27 at 11 02 24 AM" src="https://user-images.githubusercontent.com/13629122/100467238-20e96780-30a0-11eb-927f-080fa72690b7.png">
